src/doom/s_sound.c

```diff
@@ -445,6 +445,40 @@ void S_StartSound(void *origin_p, int sfx_id)
         I_Error("Bad sfx #: %d", sfx_id);
     }
 
+    // substitute missing sounds in Doom 1.2
```

Comment:
Is this actually the behavior of Doom 1.2?

If nothing else this seems like the wrong place to be putting this. Doing this in the sound code seems like a hack.

Reply:
It's not, it was just faster to code at the time. :)

I've removed this bit now, and replaced it with gameversion checks in https://github.com/chocolate-doom/chocolate-doom/commit/77f49e5ea5aae45fe904306a9364018c0fc23b71 .